service/history/ndc/resetter.go

```diff
@@ -22,7 +22,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-//go:generate mockgen -copyright_file ../../../LICENSE -package $GOPACKAGE -source $GOFILE -destination workflow_resetter_mock.go
+//go:generate go run ../../../cmd/tools/mockgen/main.go -copyright_file ../../../LICENSE -package $GOPACKAGE -source $GOFILE -destination resetter_mock.go
```

Comment:
so two generate lines were pointing at the same file? that means one of them must not have been needed, right? can we delete this instead?

Reply:
Yes, they were. Huh, good point. I thought they somehow merged. I double-checked and looks unused. Deleted.